R/qenv-extract.R

```diff
@@ -0,0 +1,48 @@
+#'
+#' @section Subsetting:
+#' `x[names]` subsets objects in `qenv` environment and limit the code to the necessary needed to build limited objects.
+#'
+#' @param names (`character`) names of objects included in `qenv` to subset
```

Comment:
in the Rd file you are writing (i.e. `@rdname qenv` -> rendered into https://insightsengineering.github.io/teal.code/main/reference/qenv.html) there is already a parameter `names` with a different description. Have you tried how this renders to a single rd file? Would it be confusing?

Reply:
Oh yes indeed. Good catch. Will try to think about it, and maybe just document in `qenv.R` (both meanings) and skip in here.

Reply:
that might be a good idea. Then the existing description would need to be generalised to be true for the methods we have there

Reply:
6338ce13a034595c19bac9c305cf607d00d65073